errors/errors.go

```diff
@@ -75,3 +76,7 @@ func InvalidEmailError(msg string, args ...interface{}) error {
 func ConnectionFailureError(msg string, args ...interface{}) error {
 	return New(ConnectionFailure, msg, args...)
 }
+
+func WrongAuthzStateError(msg string, args ...interface{}) error {
```

Comment:
Can you make this "WrongAuthorizationStateError"? Even though it's longer, it makes it easier to mentally map between the enum and the function.

Reply:
Resolved in 2d1015d - thanks!